lib/susedistribution.pm

```diff
@@ -552,7 +552,11 @@ sub activate_console {
         }
         assert_screen "text-logged-in-$user";
         $self->set_standard_prompt($user, skip_set_standard_prompt => $args{skip_set_standard_prompt});
-        assert_screen $console;
+        assert_screen([$console, "$console-with-kernel-msg"]);
```

Comment:
ok, and where does this variable come from?

Reply:
The variable is called `$console` and the string `-with-kernel-msg` is appended.

Reply:
of course, right :)

Reply:
This assert_screen is about the logged in console but weren't we talking about the login? So I assume you rather want to add tags to check for after line L535.

I am ok with the idea in general but we should try to find a test. I assume you can simulate it somehow, e.g. could you make the kernel super-verbose on purpose with boot options for testing purposes?

Reply:
It didn't work when manually booting with `loglevel=9` - kernel was still not too spammy.